lib/fog/google/models/storage_json/directories.rb

```diff
@@ -0,0 +1,37 @@
+require "fog/core/collection"
+require "fog/google/models/storage_json/directory"
+
+module Fog
+  module Storage
+    class GoogleJSON
+      class Directories < Fog::Collection
+        model Fog::Storage::GoogleJSON::Directory
+
+        def all
+          data = service.get_service.body["items"]
+          load(data)
+        end
+
+        def get(key, options = {})
+          remap_attributes(options,             :delimiter  => "delimiter",
+                                                :marker     => "marker",
+                                                :max_keys   => "max-keys",
+                                                :prefix     => "prefix")
+          data = service.get_bucket(key, options).body
+          directory = new(:key => data["name"])
+          # options = {}
```

Comment:
Looking into blame - this looks like a blind copy from AWS provider:
https://github.com/fog/fog/blame/90e96f36e6f70a12f0c6794a149231c337293887/lib/fog/google/models/storage/directories.rb
Relevant commit: https://github.com/fog/fog/blame/90e96f36e6f70a12f0c6794a149231c337293887/lib/fog/google/models/storage/directories.rb

So if it's not used - I think we can leave it out. @icco @plribeiro3000 - what do you guys think?